src/components/common/Header.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import { Link, useLocation } from 'react-router-dom';
+
+import { menu, close } from 'assets';
+
+import routes from 'routes/routes';
+
+const Header = () => {
+  const Options = [
+    {
+      title: 'About',
+      path: routes.about.path
+    },
+    {
+      title: 'Contact',
+      path: routes.contact.path
+    }
+  ];
+
+  const [optionsOpen, setOptionsOpen] = useState(false);
+
+  const location = useLocation();
+
+  const onOptionsOpen = () => {
+    setOptionsOpen((prevState) => !prevState);
+  };
+
+  return (
+    <header className="headerWrapper">
+      <button type="button" onClick={onOptionsOpen}>
+        <img
+          src={optionsOpen ? close : menu}
+          alt={optionsOpen ? 'close' : 'menu'}
+        />
+      </button>
+      {optionsOpen && (
+        <ul>
+          {Options.map((option) => (
+            <li key={option.path}>
+              <Link
+                to={option.path}
+                onClick={() => setOptionsOpen(false)}
+                className={location.pathname === option.path ? 'selected' : ''}
```

Comment:
Hmmm what about... 
```suggestion
                className={location.pathname === option.path && 'selected' }
```

Reply:
Nope, using ternary as you are suggesting returns a boolean and typescript needs a string or undefined value into className property :) 

![image](https://user-images.githubusercontent.com/41349687/165398640-d2ad5eac-8cf7-437a-9c3c-6088cb1fc6df.png)